plugin/pkg/scheduler/scheduler.go

```diff
@@ -124,7 +124,9 @@ func (s *Scheduler) scheduleOne() {
 	}
 
 	go func() {
-		defer metrics.E2eSchedulingLatency.Observe(metrics.SinceInMicroseconds(start))
+		defer func() {		
```

Comment:
Could you please explain how this change addresses the issue?
I think you should pass `start` as an argument to the go routine at line 126 to solve the problem.